JoyboyCommunity/src/components/note/PostPage.tsx

```diff
@@ -25,44 +25,4 @@ function PostPage({ navigation, post }) {
       ></TouchableOpacity> */}
     </View>
   );
-}
-
-const styles = StyleSheet.create({
```

Comment:
why move styles? i think it would be easier to make changes in the same file no?
no hard opinions tho just curious

Reply:
For me, it is easier to track styles and separate react code from style code. It is also better for split-view editing, one side react code, one side styles. But it is kinda like postcss vs tailwind. Again, i can remove it to the component's file as well if would you like.

Reply:
Oh i thought it optimises something. I think it would be cool to try this out!